src/blockly/generators/propc/communicate.js

```diff
@@ -2798,7 +2798,7 @@ Blockly.propc.oled_initialize = function () {
                 }    
             }
 
-            if (cogStartBlock && inDemo) {  // ONLY RUN IN DEMO - keep this experimental for now.
+            if (cogStartBlock && isExperimental.indexOf('volatile') > -1) {  // ONLY RUN IN DEMO - keep this experimental for now.
```

Comment:
In Solo, there is not a "Demo" system. These flags will be live when the code is pushed into an image and pushed to the ECS cluster.

Reply:
This flag has been here for a long time, when we had "DEMO", this fuse was blown - it's always worked, but we've never cone through a full validation for the generated c code...so it sits parked...not a high priority, but since it was there, thought I should put a few coins in the meter.  I'll adjust the comment to remove the reference to "DEMO".

Reply:
OK. We will want to remove all references to "Demo" in Solo since it has not application.

Reply:
I removed the reference in the comments.  I think there are some others out there, so we can just remove them as we come across them.